feat(apple): Explain accuracy of app hangs (#6110)
getsentry/sentry-docs: src/platforms/apple/common/configuration/app-hangs.mdx

There are many reasons an app can become unresponsive, from long running code to an infinite loop bug, and this can be frustrating to the end user.
With app hang tracking, you can detect and fix this problem.

> The app hang detection integration has a default timeout of two (2) seconds, which means if the app becomes unresponsive for at least two seconds, the SDK creates an error event. We improved the algorithm to detect app hangs sooner with 7.24.0:
> - Before 7.24.0 the app hang can last up to `appHangTimeoutInterval * 2` until the SDK reports an app hang.
> - Since 7.24.0 the app hang can last up to `appHangTimeoutInterval * 1.2` until the SDK reports an app hang.
> 

Reviewer: ```suggestion
The app hang detection integration has a default timeout of two (2) seconds. This means, that if the app is unresponsive for two seconds or more, the SDK creates an error event. With 7.24.0, we've improved the algorithm to detect app hangs sooner, decreasing it from `appHangTimeoutInterval * 2` to `appHangTimeoutInterval * 1.2`.

```
Let me know if this change in structure works and if it's still getting across what you're trying to say.

Author: Yep, that nails it 👏. Thanks, @lizokm.
